site/homepage/content/getting-started/index.md

```diff
@@ -35,6 +35,25 @@ The Hono command line client is used in this guide for simulating an application
 The client is available from [Hono's download page]({{< relref "downloads" >}}).
 Note that running the command line client requires a Java 11 runtime environment being installed locally.
 
+#### Very quick Quickstart
+
+In the following all necessary commands and operations will be explained on how to register a tenant and a device via the HTTP Api.
+Then, it will be shown how messages can be sent via the HTTP and MQTT Endpoint.
```

Comment:
... via the HTTP and MQTT protocol adapters.

Reply:
Changed.